tests/basic/main.rs

```diff
@@ -265,6 +265,17 @@ pub async fn async_basic_test_0() {
     )
 }
 
+#[test]
+pub fn v0_file_test() {
+    let path = basic_path("demo-11-zlib.orc");
+    let reader = new_arrow_reader_root(&path);
+    let _expected_row_count = reader.total_row_count();
+    let batches = reader.collect::<Result<Vec<_>, _>>().unwrap();
+    let _total_rows: usize = batches.iter().map(|b| b.num_rows()).sum();
+    // TODO: not reading entire file, debug
+    // assert_eq!(expected_row_count as usize, total_rows);
+}
```

Comment:
When we do collect on ArrowReader it doesn't seem to be reading all rows in the file, will need to debug separately